lnd_test.go

```diff
@@ -2000,34 +2054,67 @@ func testChannelForceClosure(net *lntest.NetworkHarness, t *harnessTest) {
 	if err != nil {
 		t.Fatalf("unable to generate block: %v", err)
 	}
-	time.Sleep(duration)
+
+	// Due to mining a large amount of blocks, we'll sleep to allow the UTXO
+	// nursery to catch up before restart.
+	time.Sleep(3 * time.Second)
```

Comment:
Heh, this took me some time to figure out... Found out it's actually because of https://github.com/lightningnetwork/lnd/pull/1608. We're expecting things to kick off at `height-1`, but because we restarted, we now wait at `height` instead. So we basically time out the `WaitPredicate` because a block should be mined but isn't. Added a TODO which can be removed in your PR.

Reply:
#1608 has been merged.